compiler/rustc_hir/src/hir.rs

```diff
@@ -3075,6 +3075,7 @@ pub enum ClosureBinder {
 pub struct Mod<'hir> {
     pub spans: ModSpans,
     pub item_ids: &'hir [ItemId],
+    pub had_parse_errors: Result<(), ErrorGuaranteed>,
```

Comment:
Why is it needed in HIR? It's never used.

Reply:
Good catch. Because it was public it didn't lint it was not being used.